src/components/Button/index.tsx

```diff
@@ -1,5 +1,10 @@
 // components/Button.jsx
-export const Button = ({ label, handleOnClick }) => {
+interface ButtonProps {
+  label: string;
+  handleOnClick: () => void;
```

Comment:
Rename to 'handleClick'. It's common ot call this function 'handleClick' or 'onClick but not combined :)